### ISSUE TITLE
Need new maintainer - can I help?

### ISSUE THREAD

**Issue by jpr5** [GpgMailNoActivation/GpgMailNoActivation#9]
Howdy, we've since added support for Catalina, and Big Sur.  People still come here looking for the right shiz..

@GpgMailNoActivation Would you consider making me or bnavigator committers to the repo so we can keep it up to date?

**Comment by DerMozartJBA** [comment]
Can you make a fork?

**Comment by HigH-HawK** [comment]
Yeah, a fork would be awesome because I think this one is dead :(

**Comment by bnavigator** [comment]
The current "active" fork is here: https://github.com/bnavigator/Free-GPGMail
Any help appreciated. I don't even own a Mac.
